python/setup.py

```diff
@@ -51,6 +51,9 @@ def _maybe_add_library_root(lib_name):
 elif sys.platform == "win32":
     cflags = ["/std:c++17", "/d2FH4-"]
     package_data["ctranslate2"] = ["*.dll"]
+elif sys.platform == "linux":
+    cflags.append("-fPIC")
+    ldflags.append("-Wl,-rpath,-Wl,-rpath,/usr/local/lib64")
 
```

Comment:
Hello! Two comments:
- -Wl,-rpath is repeated twice in " ldflags.append("-Wl,-rpath,-Wl,-rpath"
- ,/usr/local/lib64 may not work in some distros that use /usr/local/lib

A better solution may be:

ldflags.append("-Wl,-rpath,/usr/local/lib64:/usr/local/lib")